.circleci/config.yml

```diff
@@ -96,6 +96,7 @@ jobs:
       - when:
           condition: << parameters.lint >>
           steps:
+            - run: find . -path '*.podspec' -exec perl -pi -e 's/.+\.social_media_url.+//' {} \;
```

Comment:
It's a bit unfortunate if we have to allow warnings. I wonder if we can work around it by serving a dummy url locally?

Reply:
This line simply removes the social media URL but doesn’t affect other warnings that would arise. Serving a dummy URL would have exactly the same effect.